api/health_test.go

```diff
@@ -16,6 +17,10 @@ import (
 func TestHealth(t *testing.T) {
 	conf := internal.NewConfigurationWithDefaults()
 	r := internal.NewRegistry(conf)
+	r.Init()
+
+	// Waiting for rule load and health event propagation
```

Comment:
Shouldn't we check if the service is unavailable before? That way we know if the ruleloader actually works!

Reply:
easy peasy :sunglasses: